mdata/chunk/chunk.go

```diff
@@ -16,19 +15,20 @@ type Chunk struct {
 	tsz.Series
 	LastTs    uint32 // last TS seen, not computed or anything
 	NumPoints uint32
-	Saved     bool
-	Saving    bool
-	LastWrite uint32
+	Closed    bool
 }
 
 func New(t0 uint32) *Chunk {
-	// we must set LastWrite here as well to make sure a new Chunk doesn't get immediately
-	// garbage collected right after creating it, before we can push to it
```

Comment:
the equivalent of this comment should be added to AggMetric where we initialize its lastWrite field